Deploy/Systemd/ss-startup-audio.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Startup Audio For SUSI Linux
+After=multi-user.target
+
+[Service]
+ExecStart=/usr/bin/mpg123 /home/pi/SUSI.AI/susi_linux/wav/ting-ting_susi_is_alive_and_listening.wav
```

Comment:
This service is not supposed to be kept running, because the audio file is not infinite. Set service type as _oneshot_.